MAUI/DataGrid/export-to-excel.md

```diff
@@ -801,6 +801,10 @@ option.CanApplyGridStyle = true;
 
 <img alt="Export DataGrid to Excel with styling applied" src="Images\export-to-excel\maui-datagrid-can-apply-gridstyle.png" width="689"/>
 
+### Limitations
+
+* Implicit and explicit styling is not applicable.
```

Comment:
The implicit and explicit styles are not applicable when exporting the DataGrid to Excel

Reply:
Modified the content